README.md

```diff
@@ -59,4 +63,4 @@ To deploy the backend to Heroku:
 local postgres database has data and works when you run the app locally,
 then run `heroku pg:push LOCAL_DB_NAME postgres`
 
-- Visit the [application](http://text-thresher.herokuapp.com/api/) to make sure it worked.
\ No newline at end of file
+- Visit the [application](http://text-thresher.herokuapp.com/api/) to make sure it worked.
```

Comment:
good thinking. usually it will have that red no-EOL arrow if that's not present. anyhow, yeah, that's another thing that we should standardize. preferably with a config file.